### PR TITLE
develop curation view

### PR DESCRIPTION
**Pull request by magland**

### CODE REVIEW

File: .devcontainer/Dockerfile [hunk 1]
```diff
@@ -100,6 +100,10 @@ RUN pip install matplotlib mpld3
 # nano
 RUN apt-get install -y nano
 
+# unison and unison-fsmonitor
```
Comment (magland) [inline .devcontainer/Dockerfile]: Not exactly sure why this change is in this PR, but i'm not going to fret.

File: jupyterlab/labbox_ephys_widgets_jp/src/extensions/curation/CurationSortingView/AllUnitsTab.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
+import React, { FunctionComponent } from 'react';
```
Comment (magland) [inline jupyterlab/labbox_ephys_widgets_jp/src/extensions/curation/CurationSortingView/AllUnitsTab.tsx:1]: Implemented the AllUnits tab which shows the SortingView plugins. Maybe we want a better name than "All Units"

File: jupyterlab/labbox_ephys_widgets_jp/src/extensions/curation/CurationSortingView/SelectedUnitsTab.tsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
+import React, { Fragment, FunctionComponent } from 'react';
```
Comment (magland) [inline jupyterlab/labbox_ephys_widgets_jp/src/extensions/curation/CurationSortingView/SelectedUnitsTab.tsx:1]: Implemented the SlectedUnits tab which shows the SortingUnitView plugins for the selected units. Maybe we want a better name than "Selected Units"

File: jupyterlab/labbox_ephys_widgets_jp/src/extensions/curation/CurationSortingView/TabsView.tsx [hunk 4]
```diff
@@ -0,0 +1,84 @@
+import AppBar from '@material-ui/core/AppBar';
```
Comment (magland) [inline jupyterlab/labbox_ephys_widgets_jp/src/extensions/curation/CurationSortingView/TabsView.tsx:1]: Implemented this TabsView which is the right-hand window in the curation view

File: jupyterlab/labbox_ephys_widgets_jp/src/extensions/devel/IndividualUnits/IndividualUnit.tsx [hunk 5]
```diff
@@ -14,7 +14,7 @@ interface Props {
     width: number
     calculationPool: CalculationPool
     sortingInfo: SortingInfo
-    sortingUnitViews: {[key: string]: SortingUnitViewPlugin}
+    plugins: Plugins
```
Comment (magland) [inline jupyterlab/labbox_ephys_widgets_jp/src/extensions/devel/IndividualUnits/IndividualUnit.tsx:17]: I simplified and renamed how registered plugins get passed around. Prior to the change, extensionContext meant two different things -- it referred to the object passed to the extensions that could be used to register plugins, and it also referred to the redux state that contained the registered plugins. I changed the latter to 'plugins' instead. And now 'plugins' state gets passed around as opposed to separately passing around SortingUnitViewPlugins, SortingViewPlugins, etc.

File: src/containers/SortingUnitView.tsx [hunk 6]
```diff
@@ -78,7 +78,7 @@ const SortingUnitView: FunctionComponent<Props> = ({ sortingId, unitId, sorting,
             calculationPool={calculationPool}
             width={width}
             sortingInfo={sortingInfo}
-            sortingUnitViews={sortingUnitViews}
+            plugins={plugins}
```
Comment (magland) [inline src/containers/SortingUnitView.tsx:81]: Most of these changes relate to the change described in my above comment... renaming extensionContext redux state to plugins redux state, and passing plugins object rather than sortingViewPlugins, etc.

**Review by jsoules — APPROVED** [review]
Assuming we can get the graphical weirdnesses in Jupyter worked out--or that we decide to just live with it until we can reimplement everything in Canvas--this should be good to go.
